src/core/__tests__/util-test.js

```diff
@@ -283,27 +286,83 @@ const worksheetDataRowsExpected = [
 const worksheetRange = 'A1:H7';
 
 describe('createWorksheetSummaryCells', function () {
-  it('Correctly forms worksheet summary cells', function () {
-    var util = require('../util');
-    var result = util.createWorksheetSummaryCells(worksheetSummaryData, worksheetTimeOfYear);
+  it('Generates summary cells for an exportable worksheet', function () {
+    var exportfiles = require('../export');
+    var result = exportfiles.createWorksheetSummaryCells(worksheetSummaryData, worksheetTimeOfYear);
     expect(result).toEqual(worksheetSummaryCellsExpected);
   });
 });
 
 describe('fillWorksheetDataRows', function () {
-  it('Correctly fills worksheet data cells', function () {
-    var util = require('../util');
-    var result = util.fillWorksheetDataCells(worksheetTestData);
+  it('Generates data cells for an exportable worksheet', function () {
+    var exportfiles = require('../export');
+    var result = exportfiles.fillWorksheetDataCells(worksheetTestData);
     expect(result).toEqual(worksheetDataRowsExpected);
   });
 });
 
 describe('assembleWorksheet', function () {
-  it('Correctly assembles worksheet from summary and data cells', function () {
-    var util = require('../util');
-    var summaryCells = util.createWorksheetSummaryCells(worksheetSummaryData, worksheetTimeOfYear);
-    var dataCells = util.fillWorksheetDataCells(worksheetTestData);
-    var ws = util.assembleWorksheet(summaryCells.concat([[]], dataCells));
+  it('Assembles an exportable worksheet from summary and data cells', function () {
+    var exportfiles = require('../export');
+    var summaryCells = exportfiles.createWorksheetSummaryCells(worksheetSummaryData, worksheetTimeOfYear);
+    var dataCells = exportfiles.fillWorksheetDataCells(worksheetTestData);
+    var ws = exportfiles.assembleWorksheet(summaryCells.concat([[]], dataCells));
     expect(ws['!ref']).toEqual(worksheetRange);
   });
 });
+
+describe('createWorkSheetSummaryCells2', function () {
```

Comment:
All the tests that climate explorer currently has test a single function (providing an input and testing it against an expected output) and are given the same name as the function they test; I followed that convention when adding tests. There was already a test of the createWorkSheetSummaryCells function, named createWorkSheetSummaryCells.

I felt that existing test was insufficient. (Specifically, the test data used in that test, worksheetSummaryData, contains a "variable_name" attribute that isn't present in the real API data and has to be fetched separately with a metadata call. So while createWorksheetSummaryCells passed its test, exported datafiles have been giving variable name as "undefined" unnoticed for a while. This is, of course, an example of the perils of using "good" datasets, as you mentioned above.)

So I thought we needed an additional test of the createWorkSheetSummaryCells functions, and named it createWorkSheetSummaryCells2, since it also tests the createWorkSheetSummaryCells function.